build-tools/automation/azure-pipelines.yaml

```diff
@@ -0,0 +1,213 @@
+      displayName: create bundle
+      inputs:
+        solution: build-tools/create-bundle/create-bundle.csproj
+        configuration: $(XA.Build.Configuration)
+        msbuildArguments: /bl:bin/Build$(XA.Build.Configuration)/msbuild-createbundle.binlog $(MSBuildAbiArgs)
+
+    - task: CopyFiles@2
+      displayName: copy bundle
+      inputs:
+        sourceFolder: bin/$(XA.Build.Configuration)
+        contents: bundle-*.zip
+        targetFolder: $(Build.ArtifactStagingDirectory)
+
+    - task: PublishPipelineArtifact@0
+      displayName: upload bundle
+      inputs:
+        artifactName: $(BundleArtifactName)
+        targetPath: $(Build.ArtifactStagingDirectory)
+
+    - task: MSBuild@1
+      displayName: package build results
+      inputs:
+        solution: build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks.csproj
+        configuration: $(XA.Build.Configuration)
+        msbuildArguments: /t:ZipBuildStatus /p:ResultPackageOutputPath=$(Build.ArtifactStagingDirectory)/results
+      condition: always()
+
+    - task: PublishPipelineArtifact@0
+      displayName: upload artifacts
+      inputs:
+        artifactName: prepare-build-results
+        targetPath: $(Build.ArtifactStagingDirectory)/results
+      condition: always()
+
+- stage: mac_build
+  displayName: Mac
+  dependsOn: prepare
+  jobs:
+  - job: mac_build
+    displayName: Build
+    pool: Hosted Mac Internal
+    timeoutInMinutes: 240
+    cancelTimeoutInMinutes: 5
+    steps:
+    - task: DownloadPipelineArtifact@1
+      inputs:
+        artifactName: $(BundleArtifactName)
+        downloadPath: $(System.DefaultWorkingDirectory)
+
+    - script: make prepare-external-git-dependencies CONFIGURATION=$(XA.Build.Configuration)
+      displayName: make prepare-commercial
+      condition: eq(variables['XA.Commercial.Build'], 'true')
+      env:
+        GH_AUTH_SECRET: $(Github.Token)
+
+    - task: provisionator@2
+      condition: and(succeeded(), eq(variables['XA.Commercial.Build'], 'true'))
+      inputs:
+        provisionator_uri: $(provisionator-uri)
+        github_token: $(GitHub.Token)
+        provisioning_script: $(System.DefaultWorkingDirectory)/external/monodroid/build-tools/provisionator/profile.csx
+        provisioning_extra_args: -vv
+
+    - script: make prepare-deps V=1 CONFIGURATION=$(XA.Build.Configuration) MSBUILD_ARGS="$(AutoProvisionArgs)"
+      displayName: make prepare-deps
+
+    - script: make prepare jenkins V=1 CONFIGURATION=$(XA.Build.Configuration) MSBUILD_ARGS="$(AutoProvisionArgs) /p:BundleRootPath=$(System.DefaultWorkingDirectory)"
+      displayName: make prepare jenkins
+
+    - script: make create-installers V=1 CONFIGURATION=$(XA.Build.Configuration)
+      displayName: make create-installers
+
+    - script: |
+        mkdir -p $(System.DefaultWorkingDirectory)/unsigned-installers
+        cp bin/Build$(XA.Build.Configuration)/*.vsix $(System.DefaultWorkingDirectory)/unsigned-installers
```

Comment:
As with above, why copy the installers into `$topdir/unsigned-installers` instead of `$topdir/bin/Build$(XA.Build.Configuration)/unsigned-installers`?